adapters/sovrn/sovrn.go

```diff
@@ -61,6 +39,69 @@ func (s *SovrnAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 		}
 	}
 
+	errs := make([]error, 0, 1)
+	var err error
+	var validImps int = 0
+
+	for index, imp := range request.Imp {
+		var bidderExt adapters.ExtImpBidder
+		if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
+			errs = append(errs, &errortypes.BadInput{
+				Message: err.Error(),
+			})
+			continue
+		}
+
+		var sovrnExt openrtb_ext.ExtImpSovrn
+		if err := json.Unmarshal(bidderExt.Bidder, &sovrnExt); err != nil {
+			errs = append(errs, &errortypes.BadInput{
+				Message: err.Error(),
+			})
+			continue
+		}
+
+		tagId := getTagid(sovrnExt)
+		if tagId == "" {
+			errs = append(errs, &errortypes.BadInput{
+				Message: "Missing required parameter 'tagid'",
+			})
+			continue
+		}
+
+		imp.TagID = getTagid(sovrnExt)
```

Comment:
You've already gotten the `tagId` value from the `getTagid` function on L63 so this line should be just:
```
imp.TagID = tagId
```

Also, can you please rename the `getTagid` function to `getTagId`?

Reply:
Both done